src/helpers/figures/charts/chart_common.ts

```diff
@@ -35,6 +35,7 @@ import { rangeReference } from "../../references";
 import { getZoneArea, isFullRow, toUnboundedZone, zoneToDimension, zoneToXc } from "../../zones";
 
 export const TREND_LINE_XAXIS_ID = "x1";
+export const MOVING_AVERAGE_TREND_LINE_XAXIS_ID = "xMA";
```

Comment:
```suggestion
export const MOVING_AVERAGE_TREND_LINE_XAXIS_ID = "xMovingAverage";
```
nitpick. That way it's easier to know what it does if we're ever debugging a chart runtime